src/name_gen.rs

```diff
@@ -0,0 +1,15 @@
+pub fn generate_simple_name(epic_title: &str) -> String {
```

Comment:
I'm not feeling super happy about this function name. Consider another like `super_mario_epic` maybe? It's fine to leave it if you prefer this though.